### DISCUSSION TITLE
Reuse functions between hooks

### DISCUSSION THREAD

**Discussion by redtide** [mkdocs/mkdocs#3529] [Q&A] [answered]
Is it possible to reuse a function|variable defined in a `hooks1.py` in another `hooks2.py` file?

**Comment by oprypin** [comment, marked as answer]
This is tracked as a bug (+see a workaround there)

* https://github.com/mkdocs/mkdocs/issues/3281

**Reply by redtide** [reply]
My bad, wrong search keys... thank you!
